helpers/util.py

```diff
@@ -60,3 +60,27 @@ def get_used_memory():
 
 def fill_template(template, properties):
     return Template(template).substitute(properties)
+
+
+def modify_cmd(cmd: str, wrap_sh: bool = False, with_sudo: bool = False) -> str:
```

Comment:
Why do we need this method? Why can't we wrap all commands in `sh -c "cmd"`? We always run tests using `root` user or `sudo`.  I don't understand why do we needed a new flag (`with-sudo`) and `wrap_sh`.

Reply:
Discussed over a phone, updated.